### PR TITLE
edit recurrence button is aligned properly

### LINKED ISSUE #5359: "Edit Recurrence" button in Recurrence widget is not aligned properly
**Describe the bug**
The "Edit Recurrence" button present in the recurrence widget while adding or editing an event is not aligned properly.

**To Reproduce**

1. Go to 'Homepage'
2. Click on 'Add'
3. Select 'Event'
4. Scroll down to 'Recurrence'
5. See error

**Expected behavior**
The button should be properly aligned with the title "Recurrence"

**Screenshots**
image

### PR DESCRIPTION
**Pull request by Ravi-kumar9347**
Fixes #5359, In the recurrence widget, set the vertical alignment of the `edit` button to `middle`.

### CODE REVIEW

File: packages/volto/news/5359.bugfix [hunk 1]
```diff
@@ -0,0 +1 @@
+"Edit Recurrence" button in Recurrence widget is aligned properly. @Ravi-kumar9347
```
Comment (stevepiercy) [inline packages/volto/news/5359.bugfix:1]:
```suggestion
In the recurrence widget, set the vertical alignment of the `edit` button to `middle`. @Ravi-kumar9347
```

**Review by stevepiercy — CHANGES_REQUESTED** [review]
Restructure sentence for clarity.

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 please edit your issue description to comply with Item 2 in https://6.docs.plone.org/contributing/first-time.html#create-a-pull-request-from-your-fork.

I've also requested review from a couple of core contributors.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy, updated the description of issue.

**Review by stevepiercy — APPROVED** [review]
I approve only the news item. This should also be reviewed by a core contributor.

**Comment by stevepiercy** [comment]
@Ravi-kumar9347 you did not edit this description:

https://github.com/plone/volto/pull/5590#issue-2062600163

You edited a change log entry. Please carefully read and follow [my comment](https://github.com/plone/volto/pull/5590#issuecomment-1874296021).

**Comment by Ravi-kumar9347** [comment]
@sneridagh , @davisagli can you please review my pr.

**Comment by Ravi-kumar9347** [comment]
@stevepiercy now the description is correct?

**Comment by stevepiercy** [comment]
> @stevepiercy now the description is correct?

Yes, you got it.

**Review by davisagli — APPROVED** [review]
Thanks, looks good to me.

**Pull request merged**